### DISCUSSION TITLE
How to use hook in getServersideProps ?

### DISCUSSION THREAD

**Discussion by Maiveasna** [vercel/next.js#24541] [Help]
**Example :**
export const getServerSideProps: GetServerSideProps = async (ctx) => {
 
  const {  info } = userInfo()
   
    return {
        props: {
           info : info 
        }
    }
}

I want  to use hook. in. getServerProps but i still got error on my projects.

**Comment by apollonian** [comment]
Hooks can only be called from functional/stateless components. Refer the [rules of hooks](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-from-react-functions)!
